include/cityhash.h

```diff
@@ -32,6 +32,14 @@
 extern "C" {
 #endif
 
+/*
+ * We have 2/3-argument specialized versions of cityhash4,
+ * which can reduce instruction count and stack usage on some 32-bit arches.
+ * For 1-arg version, using cityhash2 is enough.
+ */
+#define	cityhash1(w) (cityhash2(w, 0))
```

Comment:
`cityhash1()` could save one multiplication if `w` as passed as the second argument.  Sure the hash will be different, but I don't think it matter, at least not for the only current use case.  Though I don't insist on it.